src/System.Management.Automation/help/CommandHelpProvider.cs

```diff
@@ -572,7 +572,7 @@ private string FindHelpFile(CmdletInfo cmdletInfo)
                 // e.g., <AssemblyName>.ni.dll as supposed to <AssemblyName>.dll.
 
                 // When cmdlet metadata is generated for the 'HelpFile' field, we use the name assembly and we append '-Help.xml' to it.
-                // Because of this, if the cmdlet is part of an N�gen assembly, then 'HelpFile' field will be pointing to a help file which does not exist.
+                // Because of this, if the cmdlet is part of an N'gen assembly, then 'HelpFile' field will be pointing to a help file which does not exist.
```

Comment:
Ngen refers to Ngen.exe which produces native code from managed code, so the apostrophe doesn't make sense